packages/ivh-base-components/src/molecules/select/select-menu-item.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { AuListItem } from '@ivantagehealth/ivh-base-components'
+import { AuListItem } from '../../atoms'
```

Comment:
I'm a little surprised that this was necessary. It's possible that was just an artifact of developing locally with linked dependencies. Our webpack config sets `@ivantagehealth/ivh-base-components` as an external and so doesn't reach in and grab an `AuListItem` during build when referenced from the named import. At runtime, I was getting errors about the list item HOC getting undefined for `Cmp` when using an `AuSelectMenuItem.Link`. Moving over to reference `AuListItem` from the atoms file directly side stepped the issue.